src/main/daml/Daml/Finance/Interface/Lifecycle/Rule/Claim.daml

```diff
@@ -44,7 +44,7 @@ interface Settlement where
     do
       pure $ _view this
 
-  nonconsuming choice Claim : ClaimResult
+  nonconsuming choice Effectuate : ClaimResult
```

Comment:
Maybe `Process` is an alternative to `Effectuate`? Ie., you process a claim and get a `ClaimResult`.

Reply:
changed to `ClaimEffect` for now.